### ISSUE TITLE
How to start perl-toolbox in VS Code?

### ISSUE THREAD

**Issue by kwein1** [d9705996/perl-toolbox#23]
I'm new to using extensions in VS Code. How does one start Perl-Toolbox to run its lint against some Perl code in VS Code?
When I type F5, Perl-Toolbox is not an option.

**Comment by dirk-seynhaeve** [comment]
TL;DR: it works out of the box, while you're editing Perl source code.

### Explanation
Perl Toolbox is a "Problem Matcher".

### Tasks

- Compiling source code is considered a "**Build Task**", managed by a `task.config` file.

- Debugging/running executable is considered a "**Launch Task**", managed by a `launch.json` file (and this is initiated by the `F5` hot key).

- A **Problem Matcher** works on files with a specific extension, while editing them. This is managed through a `task.jso`n file, or handled automatically through the extension (which is the case for the Perl Toolbox)

You might have to bring up the `PROBLEMS` console (`Ctrl+Shift+M`) to see the results of the Problem Matcher.  

### Potential problems
Perl Toolbox uses (but does _not_ install):

- Linting: perlcritic, perltidy
- Syntax: perl

This means that Perl Toolbox needs those tools to be installed, and accessible:

- On Linux, if those tools are not in your PATH, you need to provide the location in the Perl Toolbox settings.
- On Windows/WSL, I believe that you always have to update the settings with the correct location/name of the auxiliary applications (I know I did, but it has been a while so maybe the situation has changed).

**Comment by Bu11etmagnet** [comment]
A problem matcher is a parser for task output

https://code.visualstudio.com/Docs/editor/tasks#_processing-task-output-with-problem-matchers

perl-toolbox doesn't seem to provide one. When editing tasks.json, 

        "problemMatcher": [ | ],

nothing vaguely resembling perl-toolbox appears in the suggested list of problem matchers:
![image](https://user-images.githubusercontent.com/7579651/114406672-3f578780-9b97-11eb-857f-b418d92705c6.png)

**Comment by davehodg** [comment]
I'm with kwein. I have the module installed, I have perlcritic installed. How do I get them talking? It's not out of the box. I also have no idea how to manipulate the variables mentioned in the docs.

As an aside, where do VSC people hang out on the internet, so I can bug them instead? A reference to a doc on running plugins and setting variables would be useful.
